spec/lib/pre_assembly/digital_object_spec.rb

```diff
@@ -7,7 +7,9 @@
 
   let(:pid) { 'druid:gn330dv6119' }
   let(:stager) { PreAssembly::CopyStager }
-  let(:bc) { create(:batch_context, staging_location: 'spec/fixtures/images_jp2_tif') }
+  let(:bc) { create(:batch_context, staging_location: 'spec/fixtures/images_jp2_tif', content_structure:, processing_configuration:) }
+  let(:processing_configuration) { 'filename' }
+  let(:content_structure) { 'simple_image' }
   let(:job_run) { create(:job_run, :preassembly, batch_context: bc) }
```

Comment:
changes in this spec to ensure we always use valid combination of content type and processing configuration and to better setup the batch context instead of mocking values later